_includes/tutorials/error-handling/confluent/markup/dev/run-consumer.adoc

```diff
@@ -0,0 +1,36 @@
+////
+  This is a sample content file for how to include a console consumer to the tutorial, probably a good idea so the end user can watch the results
+  of the tutorial.  Change the text as needed.
+
+////
+
+Now that you've ran the Kafka Streams application, it should have shut it self down due to reaching the max-error threshold.
+
+Let's now run the Confluent Cloud CLI to confirm the output:
+
+```
+ccloud kafka topic consume output-topic --from-beginning
+```
+
+Your results should look something like this:
+++++
+<pre class="snippet"><code class="shell">
+ALL
+ALL
+STREAMS
+LEAD
+TO
+CONFLUENT
+ALL
+STREAMS
+LEAD
+TO
+CONFLUENT
+GO
+Processed a total of 12 messages
```

Comment:
This doesn't stop at 12, does it?  It should be `ctrl-c` to stop.

Reply:
Yup, it's there on line 36 :) Do you want me to move it up?

Reply:
Great catch! There was already `crtl-c` exit text but I failed to remove the "Processed a total of 12 messages". I removed that text in commit https://github.com/confluentinc/kafka-tutorials/pull/854/commits/e33b3de9bc1b607c46a6a9827339ef5ce353645e